djangoproject/static/js/fundraising-heart.js

```diff
@@ -45,48 +45,50 @@
     { x: 3, y: 5, color: razzmatazz },
   ];
 
-  var Heart = function (selector) {
+  const Heart = function (selector) {
     this.heart = document.querySelector(selector);
     this.init();
   };
 
   Heart.prototype = {
     init: function () {
-      this.pixels = pixels.map(function (pixel) {
+      this.pixels = pixels.map((pixel) => {
         return new Rectangle(pixel);
       });
       this.fadePixels();
       this.draw();
-      var heart = this;
+      const heart = this;
     },
     fadePixels: function () {
-      var pixels;
-      var percent = this.heart.dataset.percent;
-      var fadedCount = Math.ceil((this.pixels.length * (100 - percent)) / 100);
-      for (var i = 0; i < fadedCount; i++) {
+      const pixels;
```

Comment:
pixels is reassigned later.
```suggestion
      let pixels;
```